### PR TITLE
Clarify feed is and Subject

### PR DESCRIPTION
**Pull request by SteveLasker**
Fixes accidental merger and reversion of #165
See #165 for the discussion

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -218,7 +218,9 @@ Equivocation:
 
 Feed:
 
-: see Subject
+: A collection of Signed Statements, observed by the Transparency Service, related to the `sub` field of the 'CWT_Claims' property of the protected header of Signed Statements.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:221]:
```suggestion
: An unordered collection of arbitrary data (artifacts, signed statements, receipts, or other data) related to the text identifiers, observed by the transparency service in the `sub` field of the CWT Claim property of the protected header of Signed Statements.
```

**Review by OR13 — CHANGES_REQUESTED** [review]
Repeating my change request from https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/165#discussion_r1471449646

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -218,7 +218,9 @@ Equivocation:
...
+
+Verifiers use the Feed to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the Artifact they are evaluating.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:223]:
```suggestion
```

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -262,10 +264,9 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: a logical collection of Statements about the same Artifact.
-For any step or set of steps in a supply chain there may be multiple statements made about the same Artifact.
-Issuers use Subject to create a coherent sequence of Signed Statements about the same Artifact and Verifiers use the Subject to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.
-In SCITT, Subject is a property of the dedicated, protected header attribute `15: CWT_Claims` within the protected header of the COSE envelope.
+: This term has the same definition as in RFC8392.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:267]:
```suggestion
: This term has the same definition as in RFC8392, which relies on the definition in RFC7519, which is repeated here with some modifications for readability: The "sub" (subject) claim identifies the principal that is the subject of the CWT.  The claims in a CWT are normally statements about the subject.  The subject value MUST either be scoped to be locally unique in the context of the issuer or be globally unique. The processing of this claim is generally application specific.  The "sub" value is a case-sensitive string containing a StringOrURI value.```

Comment (OR13) [reply draft-ietf-scitt-architecture.md:267]: It seems essential to repeat this definition based on the discussions we have had regarding identifiers and their duration.

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md:223]: A feed returns the collection of signed statements (or transparent statements). It returns what's on the append-only log. 
It simply returns what the transparency service persisted.

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md:221]: Unordered: 
This isn't necessarily so. A transparency service may order them. They may even order by different parameters, which is implementation specific. Stating they're not ordered is an over-statement.
This looks like a lot of extra text, to demonstrate we really need to solve what gets persisted, and what gets returned.

Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md:221]: And, this same text has to be fixed whether it's on the feed or the subject property.

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md]: done

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md]: See https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/176/commits/e674b8c1c9b19dbbbe8fc75dd4ac993670438e3d for clarification 

> Feed:
>
> : A collection of Receipts, as recorded by the Transparency Service, based on filtering of properties from the envelope including, but not limited to the `sub` field of the `CWT_Claims`.

**Review by OR13 — APPROVED** [review]
We still need to explain what kind of interoperability is expected for feed.

Does every TS need to index the same way? 

Does every TS have to support the same feed identifiers?

Are there query parameters that must always be implemented consistently?

The more I think about this, the more I feel feed does not belong in the architecture.

**Comment by SteveLasker** [comment]
Addressing @OR13's feedback, yes. We should capture those details, likely in SCRAPI. 
I've updated the PR to say it returns receipts, which defines the structure. I'd suggest the parameters and filters are part of the API.

With minimal edits, as this PR shuffled wording from Subject to Feed, I'm going to merge so we can focus on the larger changes in #179

**Pull request merged**